prow/cluster/starter.yaml

```diff
@@ -434,7 +434,7 @@ spec:
           serviceName: hook
           servicePort: 8888
 ---
-apiVersion: apps/v1
+apiVersion: extensions/v1beta1
```

Comment:
Do we have a recommended version of k8s to use? This is not necessary on recent releases AFAICT

Reply:
not completely sure, I just noticed this same apiVersion applied to the rest of the deployments in this yaml, so I'm thinking whatever version is used it should be the same in all within this starter.yaml?

Reply:
Yeah, that makes sense